### DISCUSSION TITLE
Tested the extension

### DISCUSSION THREAD

**Discussion by ghost** [goosepirate/lox365#16] [General]
The idea is splendid because Microsoft's IMAGE function in Excel doesn't work with local pictures or with pictures from LAN shares. That's why I was hoping to find a better alternative with LibreOffice and LOX365. This also because the official LibreOffice account on Mastodon presented the extension here: https://fosstodon.org/@libreoffice/111052323463423962. 

My device and version: Version: 7.5.6.2 (X86_64) / LibreOffice Community
Build ID: f654817fb68d6d4600d7d2f6b647e47729f55f15
CPU threads: 8; OS: Windows 10.0 Build 22621; UI render: Skia/Vulkan; VCL: win
Locale: de-CH (de_CH); UI: de-DE
Calc: CL threaded

The test of the extension showed the following:
General: I'm a bit sad that you have to write the formula in a cell to place the image in another cell. Furthermore, the image does not adjust to the cell size (unlike Excel's IMAGE function). And you can't provide height or width within the formula.

Using an image URL: With URLs it works. The image appears, albeit with the limitations mentioned above. 

Using a local file: The image appears only if it is in the same folder as the Calc file. A reference to another folder (using path) shows no image, although the green checkmarks in the function assist show that the path is correct.

Using a LAN share: Same here. Although the path is correct (green check marks in the function assist), the image does not appear in the Calc file.

Can the community help in solving these problems? Maybe these issues can only be fixed if it's a built-in function in Calc.

**Comment by goosepirate** [comment]
Hi, thanks for your interest. Yes, the cell reference to the output location is required, this is to my knowledge a limitation in LibreOffice's scripting API. I'm sure a built-in function would not require it.

Adjusting the image size and referencing other folders are technically possible. You're welcome to contribute code if you want this feature.
